source/module_cell/klist.h

```diff
@@ -29,60 +102,121 @@ class K_Vectors
     K_Vectors();
     ~K_Vectors();
 
-    void set(
-        const ModuleSymmetry::Symmetry &symm,
-        const std::string &k_file_name,
-        const int& nspin,
-        const ModuleBase::Matrix3 &reciprocal_vec,
-        const ModuleBase::Matrix3 &latvec);
+    void set(const ModuleSymmetry::Symmetry &symm,
+             const std::string &k_file_name,
+             const int& nspin,
+             const ModuleBase::Matrix3 &reciprocal_vec,
+             const ModuleBase::Matrix3 &latvec);
 
-    void ibz_kpoint(const ModuleSymmetry::Symmetry &symm, bool use_symm,std::string& skpt, const UnitCell &ucell, bool& match);
+    void ibz_kpoint(const ModuleSymmetry::Symmetry &symm, 
+                    bool use_symm,std::string& skpt, 
+                    const UnitCell& ucell, 
+                    bool& match);
     //LiuXh add 20180515
-    void set_after_vc(
-            const ModuleSymmetry::Symmetry &symm,
-            const std::string &k_file_name,
-            const int& nspin,
-            const ModuleBase::Matrix3 &reciprocal_vec,
-            const ModuleBase::Matrix3 &latvec);
+    void set_after_vc(const ModuleSymmetry::Symmetry &symm,
+                     const std::string &k_file_name,
+                     const int& nspin,
+                     const ModuleBase::Matrix3 &reciprocal_vec,
+                     const ModuleBase::Matrix3 &latvec);
     //get global index for ik
     inline int getik_global(const int& ik) const;
 
-    
-    int get_nks() const
-    {
-        return this->nks;
-    }
+// REFACTORING K_Vectors functions below >>>
```

Comment:
I don't think this annotation can help developers because distinguishing between old and new functions will breaks the integrity of the class.